src/pages/index.tsx

```diff
@@ -8,12 +8,14 @@ const Home: React.FC = () => {
   if (status === 'authenticated') {
     // Then redirect the user to the app pages
   }
-
+  console.log(session);
   return (
     <>
       <Header />
       <LandingContent />
       <Footer />
+      <button onClick={() => signOut()}>sign Out</button>
+      <button onClick={() => signIn()}>Login</button>
```

Comment:
redirect to login page

Reply:
The signIn( ) function with no arguments will automatically direct users to the login page. Also, the Header component has signIn( ) and signOut( ) written in it, so I'll remove these 2 lines.